mail/astroid/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        astroidmail astroid 0.15 v
+
+description         Astroid Mail
+long_description    A graphical threads-with-tags style, lightweight and fast, e-mail client for Notmuch
+maintainers         @arietis \
+                    openmaintainer
+categories          mail
+platforms           darwin
+homepage            https://astroidmail.github.io/
+license             GPL-3
+
+checksums           rmd160  a7babb6609724a7b8fe950bc1321ee61b2af7609 \
+                    sha256  0eb8a5d9d6f48eb79a64a35e7dc0919ee0d20c3104c0f7b2c2d185c6ec24df34 \
+                    size    3365848
+
+depends_build       port:boost \
+                    port:cmake \
+                    port:pkgconfig \
+                    port:protobuf3-cpp \
+                    port:scdoc
+
```

Comment:
Ports specified in `depends_build` are understood to be components only required to build the project, and not needed afterwards.

`boost` and `protobuf3-cpp` don't fall into this category.

They are libraries, so these will be needed by `astroid` whenever `astroid` runs.

So `boost` and `protobuf3-cpp` need to move to `depends_build` - which means that not only are they needed to _build_ the software, they're also needed to _run_ the software.